runtime/indent/systemverilog.vim

```diff
@@ -40,6 +44,12 @@ function SystemVerilogIndent()
     let indent_modules = 0
   endif
 
+  if exists('b:systemverilog_indent_ifdef_off')
+    let indent_ifdef = 0
```

Comment:
Were these supposed to be script-local?

Reply:
They are local to the function, and it looks like that's OK.

Reply:
Sorry for the noise, complete human diff parser failure.

@bram, if you haven't already done so, can you please remove my name from the last change list now that @kocha has the change?